app/web/package.json

```diff
@@ -22,6 +22,7 @@
     "react-dom": "^17.0.2",
     "react-error-boundary": "^3.1.3",
     "react-gtm-module": "^2.0.11",
+    "react-helmet": "^6.1.0",
```

Comment:
This is no longer maintained. How about [`react-helmet-async`](https://github.com/staylor/react-helmet-async)? It's a few extra lines but probably a safer option (until the NYT decides to stop maintaining it 😓)

Reply:
migrated to `react-helmet-async`